web/models/queries.ex

```diff
@@ -56,5 +79,25 @@ defmodule Entice.Web.Queries do
       _        -> {:error, :database_inconsistent}
     end
   end
-  
+
+  def get_friendlist(id) do
```

Comment:
I think this is not needed, since we can pre-fetch all friends that come with an account when we fetch the account itself. Its done in a similar fashion with characters of an account. But in oder to make that happen we'll have to add this information to the account model i think.